packages/reporting-server/README.md

```diff
@@ -51,7 +48,7 @@ by default it uses a in-memory sqlite instance, to use other databases, install
 pip3 install reporting-server[postgres]
 ```
 
-Then in your config, set the `db_url` accordingly, the url should be in the form
+Create the database instance on the engine that you chose. Then in your config, set the `db_url` accordingly, the url should be in the form
```

Comment:
Now that it is no longer possible to `npm start` without a postgres instance, it might be a good idea to add a section to help people set up the development environment.

Reply:
Makes sense. I'll add that. Thanks